index.html

```diff
@@ -2,14 +2,15 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Startrly</title>
+    <link href="styles/normalize.css" rel="stylesheet">
+    <link href="styles/styles.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Great+Vibes" rel="stylesheet">    <title>Startrly</title>
```

Comment:
I love how you added google fonts to further customize your site. Your quotes specifically look really great!